sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2127,5 +2127,36 @@ ALTER TABLE `form_bronchitis` MODIFY `diagnosis3_bronchitis_form` text;
 ALTER TABLE `form_bronchitis` MODIFY `diagnosis4_bronchitis_form` text;
 #EndIf
 
+#IfColumn patient_data care_team
+ALTER TABLE `patient_data` DROP COLUMN `care_team`;
+#EndIf
+
+#IfMissingColumn patient_data care_team_provider
+ALTER TABLE `patient_data` ADD COLUMN `care_team_provider` int(11) DEFAULT NULL;
+#EndIf
+
+#IfMissingColumn patient_data care_team_facility
+ALTER TABLE `patient_data` ADD COLUMN `care_team_facility` int(11) DEFAULT NULL;
+#EndIf
+
+#IfRow2D layout_options form_id DEM field_id care_team
+DELETE FROM `layout_options` WHERE form_id='DEM' AND field_id='care_team';
+UPDATE `layout_options` SET seq='14' WHERE form_id='DEM' AND field_id='cmsportal_login';
+UPDATE `layout_options` SET seq='15' WHERE form_id='DEM' AND field_id='imm_reg_status';
+UPDATE `layout_options` SET seq='16' WHERE form_id='DEM' AND field_id='imm_reg_stat_effdate';
+UPDATE `layout_options` SET seq='17' WHERE form_id='DEM' AND field_id='publicity_code';
+UPDATE `layout_options` SET seq='18' WHERE form_id='DEM' AND field_id='publ_code_eff_date';
+UPDATE `layout_options` SET seq='19' WHERE form_id='DEM' AND field_id='protect_indicator';
+UPDATE `layout_options` SET seq='20' WHERE form_id='DEM' AND field_id='prot_indi_effdate';
+#EndIf
+
+#IfNotRow2D layout_options form_id DEM field_id care_team_provider
+INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`,`fld_rows`) VALUES ('DEM', 'care_team_provider', '3', 'Care Team (Provider)', 21, 11, 1, 0, 0, '', 1, 1, '', '', '', 0);
+#EndIf
+
+#IfNotRow2D layout_options form_id DEM field_id care_team_facility
```

Comment:
ok if skip a seq so no need for above 2 code blocks. In this if would do following:
1. care_team field_id to care_team_provider
2. find the highest seq in the group_id that the care_team_provider field_id is in (also get the group_id)
3. Then change the seq in care_team_provider to this max seq + 1
4. Then add the care_team_facility entry with max seq + 2 and with the group_id obtained in step 2

Reply:
Doing above allows addition of new fields even if their group_id's have been changed with customization

Reply:
Also note that the care_team_provider and care_team_facility will need to support multiselect (likely means need new datatypes that adapted from 11 and 35)

Reply:
Did you mean user should able to select multiple providers?

Reply:
yep, for the care team provider and care team facility, should be able to add multiple selections. can see how this is done for the multiselect list datatype which basically just extended the normal select list datatype.

Reply:
Do we need to create new datatype for multi-select and how will we store multiple id in one field?

Reply:
Would make them TEXT columns in database and can check out data_type 36 to see how it stores and collects the items from one field in library/options.inc.php

Reply:
but data_type 36 accepts data from `list_options` only and we require data from `facility`  & `users` table :thinking:

Reply:
Should we make a util function which gets array of data and converts it into Multiple-Select list and Simple Select List?

Reply:
if compare the difference between data_type 36 and 1, then will note:
1. having `multiple='multiple'` in the select
2. take the sent array and convert it to a string with `|` delimiter
3. store the elements in the database field with the `|` delimiter

So, basically need to duplicate the 11 and 25 date_type into new data_types that support 1 - 3 above